test/mobile_app_backend_web/channels/predictions_for_stops_v2_channel_test.exs

```diff
@@ -32,10 +32,10 @@ defmodule MobileAppBackendWeb.PredictionsForStopsV2ChannelTest do
 
     response = %{
       predictions_by_stop: %{
-        "12345" => %{"p_1" => prediction_1, "p_2" => prediction_2},
-        trips: %{"trip_1" => trip_1, "trip_2" => trip_2},
-        vehicles: %{"v_1" => vehicle_1, "v_2" => vehicle_2}
-      }
+        "12345" => %{"p_1" => prediction_1, "p_2" => prediction_2}
+      },
+      trips: %{"trip_1" => trip_1, "trip_2" => trip_2},
+      vehicles: %{"v_1" => vehicle_1, "v_2" => vehicle_2}
```

Comment:
This test data had the wrong shape, and that could be confusing.

Reply:
Thanks for fixing!